dprint.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do you remember why it was necessary to have a dprint config file outside of the eslint config definition? Are we using dprint as a standalone ever (for markdown/json?)

Reply:
Uhhhhh, I think maybe I forgot to remove it when I changed to the eslint version?

Reply:
Hmm, okay; sounds fair! I just tested this myself and didn't need it, so I was curious. I couldn't find a way around the line-width unfortunately, I don't think there's a way given how it operates. So, overall this LGTM!